charts/argo/Chart.yaml

```
@@ -5,13 +5,9 @@ name: argo
 version: 0.16.10
```

We also bumped to a new major when deprecating argo-ci, so i would do the same here

```suggestion
version: 1.0.0
```